root.spec

```diff
@@ -22,6 +22,21 @@ Requires: freetype
 
 %define keep_archives true
 
+%define subpackageDebug true
+
+%if "%{?subpackageDebug:set}" == "set"
+%define envOptions USER_CXXFLAGS="-fdebug-prefix-map=%{cmsroot}=%{installroot} -fdebug-prefix-map=%{instroot}=%{installroot} -g -gdwarf-2 -O0"
```

Comment:
This would disable all optimisation in ROOT, making it dead slow. I would also drop "-gdwarf-2", it should be ancient already. 4 is default with 5 support coming in. Some flags like `-fvar-tracking-assignments` requires at least DWARF4. All our debuggers are new and should not have issues with DWARF4.